package.json

```
@@ -4,19 +4,41 @@
   "description": "A simple blog project",
   "main": "index.js",
   "scripts": {
-    "start": "node index.js",
+    "start": "nodemon -x ts-node main.ts",
```

_start_ script can't be .ts
```suggestion
    "dev": "tsc --watch",
    "start": "node ./dist/main.js",
```